components/connector-service/internal/externalapi/managementinfohandler_test.go

```diff
@@ -146,4 +155,103 @@ func TestManagementInfoHandler_GetManagementInfo(t *testing.T) {
 		require.NoError(t, err)
 		assert.Equal(t, http.StatusInternalServerError, rr.Code)
 	})
+
+	t.Run("should successfully get certificate info for application", func(t *testing.T) {
+		//given
+		applicationContext := clientcontext.ApplicationContext{
+			Application: appName,
+			ClusterContext: clientcontext.ClusterContext{
+				Tenant: tenant,
+				Group:  group,
+			},
+		}
+
+		extApplicationCtx := &clientcontext.ExtendedApplicationContext{
+			ApplicationContext: applicationContext,
+			RuntimeURLs: clientcontext.RuntimeURLs{
+				MetadataURL: "https://metadata.base.path/application/v1/metadata/services",
+				EventsURL:   "https://events.base.path/application/v1/events",
+			},
+		}
+
+		connectorClientExtractor := func(ctx context.Context) (clientcontext.ClientContextService, apperrors.AppError) {
+			return *extApplicationCtx, nil
+		}
+
+		commonName := extApplicationCtx.GetCommonName()
+		expectedCertInfo := certInfo{
+			Subject:      fmt.Sprintf("OU=%s,O=%s,L=%s,ST=%s,C=%s,CN=%s", organizationalUnit, organization, locality, province, country, commonName),
```

Comment:
Please, move it to the code where other variables (e.g. application, tenant) are initialized.

Reply:
Done!